### ISSUE TITLE
i18n Time partially translated in timeline view

### ISSUE THREAD

**Issue by Quent-in** [tootsuite/mastodon#3399]
When browsing someone's profile the "XX minutes ago" is properly translated but when one hovers it the date is not translated.
I would like to ask what component does this job so I can make a translation for this.
See here : [https://framapic.org/b7FLZygcoiHT/dwS6H6V6rxxZ.png](https://framapic.org/b7FLZygcoiHT/dwS6H6V6rxxZ.png)

* * * *

-  I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by nolanlawson** [comment]
Is this a bug in the translation or the component? What language is your screenshot from?

BTW this particular component is most likely [relative_timestamp.js](https://github.com/tootsuite/mastodon/blob/8963f8c3c2630bfcc377a5ca0513eef5a6b2a4bc/app/javascript/mastodon/components/relative_timestamp.js).

**Comment by Quent-in** [comment]
Hello,
is issue is that the name of the month is in English whereas I'm using Mastodon is Occitan.
Where could I add the vocabulary for the name of the months?

**Comment by ykzts** [comment]
@Quent-in 

edit "`config/locales/oc.yml`".

```console
$ git diff
diff --git a/config/locales/oc.yml b/config/locales/oc.yml
index 7f6e3daf..bea06858 100644
--- a/config/locales/oc.yml
+++ b/config/locales/oc.yml
@@ -208,6 +208,21 @@ oc:
     follow: Sègre
     prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
     title: Sègre %{acct}
+  date:
+    abbr_month_names:
+      -
+      - 1
+      - 2
+      - 3
+      - 4
+      - 5
+      - 6
+      - 7
+      - 8
+      - 9
+      - 10
+      - 11
+      - 12
   datetime:
     distance_in_words:
       about_x_hours: "Fa %{count} oras"
$ RAILS_ENV=production ./bin/rails r "I18n.locale = :oc; puts I18n.l(Time.now)"
Creating scope :cache_ids. Overwriting existing method Notification.cache_ids.
5 29 2017 a 16o16
```

**Comment by Quent-in** [comment]
So this :+1: ` abbr_month_names:
      -
      - genièr
      - febrièr
      - març
      - abrial
      - mai
      - junh
      - julhet
      - agost
      - setembre
      - octòbre
      - novembre
      - decembre`
would be correct?

**Comment by Quent-in** [comment]
@ykzts I understand that I have to add **date:** and **abbr_month_names:**
But the list of months you gave doesn't have ":" at the end, which I find strange.
Could you help me with this?

**Comment by ykzts** [comment]
see https://github.com/svenfuchs/rails-i18n/blob/cebc19038c0402e97bc6d47aabab44c1045428c2/rails/locale/en.yml#L19

**Comment by Quent-in** [comment]
Now solved with #3557
Many thanks
